ironfish-cli/src/commands/service/stats.ts

```diff
@@ -50,6 +50,7 @@ export default class Stats extends IronfishCommand {
     await this.sdk.client.connect()
 
     // metric loops, must await last loop
+    await this.chainDBSize(api, flags.delay)
```

Comment:
I think this will have to be `void` so it doesn't block the other loops (since it's a `while(true)`)